src/controllers/MakeController.php

```diff
@@ -154,7 +154,7 @@ private function makeZip(): Response
         $Response->headers->set('X-Accel-Buffering', 'no');
         $Response->headers->set('Content-Type', 'application/zip');
         $Response->headers->set('Cache-Control', 'no-store');
-        $contentDisposition = $Response->headers->makeDisposition(ResponseHeaderBag::DISPOSITION_ATTACHMENT, $Make->getFileName());
+        $contentDisposition = $Response->headers->makeDisposition(ResponseHeaderBag::DISPOSITION_ATTACHMENT, $Make->getFileName(), 'elabftw-default-filename');
```

Comment:
maybe something like "elabftw-export.zip" would be better here.

Reply:
Sure, as it’s only for zip downloads. Makes sense, I will set this name.